tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "compilerOptions": {
     "moduleResolution": "node",
     "target": "es5",
+    "downlevelIteration": true,
```

Comment:
why do we need this? Do we need to mention the following point from the TS docs?

> Please note that this requires a native Symbol.iterator or Symbol.iterator shim at runtime for any non-array values

Should we include such a shim as part of this package?

Reply:
This is because the target is ES5 but we are using iterators in one or two spots (via for of loops). I don't think we need to include anything here as our supported browsers all have `Symbol`.